src/server/sv_main.c

```diff
@@ -84,7 +84,7 @@ cvar_t *sv_wwwFallbackURL; // URL to send to if an http/ftp fails or is refused
 
 cvar_t *sv_cheats;
 cvar_t *sv_packetloss;
-cvar_t *sv_packetdelay;
+//cvar_t *sv_packetdelay;
```

Comment:
Why? What happened to that cvar?

Reply:
It is declared in src/qcommon/common.c line 131 and conflict with etlegacy src/server/sv_main.c if I remember well.